WooCommerce/WooCommerceUITests/Screens/Settings/SettingsScreen.swift

```diff
@@ -27,7 +27,7 @@ final class SettingsScreen: BaseScreen {
     }
 
     @discardableResult
-    func logOut() -> PrologueScreen {
+    func logOut() throws -> PrologueScreen {
```

Comment:
Nice – it makes sense to have `throws` be applied to any screen method – every single one has to perform IO (to the XCUITest framework).

And no more `try!` 😁